setup.py

```diff
@@ -35,21 +34,24 @@
             ],
         'test': [
             'nose',
+            'flake8',
             'openfisca-dummy-country == 0.1.6',
             ],
         },
     include_package_data = True,  # Will read MANIFEST.in
     install_requires = [
-        'Babel >= 0.9.4',
         'Biryani[datetimeconv] >= 0.10.4',
         'numpy >= 1.11',
         'PyYAML >= 3.10',
+        'flask == 0.12',
```

Comment:
Is this just a suggestion or a required change?

I thought about it, but it makes everything more complex and I don't see the real use case. The main drawback here is that we make the installation heavier, but the coast is quite low.